lib/adapter.js

```diff
@@ -377,7 +377,7 @@ function createMessageListener() {
 function createProbeListener(probeName) {
   monitor.on(probeName, function(res) {
```

Comment:
I think the argument name is wrong here, this is the `data` object from the `express` even that had docs removed in https://github.com/RuntimeTools/appmetrics/pull/260/files, right? `res` makes it seem like it is a https://nodejs.org/docs/latest/api/http.html#http_class_http_serverresponse

But the express probe got removed, and replace with strong-express-metrics... and now we are hitting this, which means that there is still an `express` event, what is the structure of this event?